recipes/pandas-vet/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pandas-vet" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57ea4906c176d61ea9c186607be7a53f4f1561b53f19f5b76ce2a5050f0bad41
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - attrs
+    - flake8 >3.0.0
```

Comment:
Is this a host dep? (also attrs)

Reply:
I am new to conda-forge and used `conda skeleton` to build the `meta.yml`, so I am not totally sure...but this package is a linter plugin for flake8, so it does need that to function. flake8 is not just for development of the package.

Reply:
The host section is for packages which must be installed during the building. Most python packages only have a host dependence on python and pip.

Reply:
Thanks for catching this and pointing me in the right direction, @CJ-Wright . Looks like it builds successfully with the extraneous packages removed. Anything else I need to clean up?